config/locales/es.yml

```diff
@@ -92,6 +92,7 @@ es:
       accept: "aceptar"
       reject: "rechazar"
       restart: "reanudar"
+      release: "release"
```

Comment:
Is it "release" in Spanish?

Reply:
`lanzamiento` don't seems so right, but okay.